File: 461.HammingDistance/distance.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        z = x ^ y
        d = 0
        while z:
            d += z % 2
            z /= 2
        return d

if __name__ == '__main__':
    print Solution().hammingDistance(1,4)

File: 461.HammingDistance/Distance.java
public class Distance {
    public int hammingDistance(int x, int y) {
        int z = x ^ y;
        int d = 0;
        while (z != 0) {
            d += z % 2;
            z /= 2;
        }
        return d;
    }

    public static void main(String[] args) {
        System.out.println(new Distance().hammingDistance(1,4));
    }
}
